### PR TITLE
Update README.md installation notes

### PR DESCRIPTION
**Pull request by dhalman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,24 @@ Installing by compilation is very straight forward. The following dependencies a
 - Go
 - GCC
 
-### Method 1 - Download and install
+### Method 1 - Fresh Install
+If you need to install vroomy:
 ```bash
-curl -s https://raw.githubusercontent.com/vroomy/vroomy/master/install | bash -s
```
Comment (itsmontoya) [inline README.md:13]: Why can't we just update the script?

**Review by itsmontoya — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (dhalman) [reply README.md:13]: bin/install is now a compile -X(flags) command that can't run from inside go

we can add a new script if you like?

Comment (dhalman) [reply README.md:13]: added a new script :)

File: utils.go [hunk 2]
```diff
@@ -36,6 +36,12 @@ func setupRuntime() (cmd *flag.Command) {
 		handleError(err)
 	}
 
+	if customCfg := cmd.StringFrom("config"); customCfg != "" {
+		fmt.Println("CustomConfig: " + customCfg)
+		configLocation = customCfg
+		cfg, cfgErr = service.NewConfig(configLocation)
```
Comment (itsmontoya) [inline utils.go:42]: Can we handle this `cfgErr` just in case?

**Review by itsmontoya — CHANGES_REQUESTED** [review]

File: utils.go (hunk 2 above)
Comment (dhalman) [reply utils.go:42]: we do, but not immediately (in case someone is using a command that does not require a config

Comment (dhalman) [reply utils.go:42]: it's handled on line 59 :)

File: bin/init [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd ~/
+
+go get github.com/hatchify/git-tagger
+go install github.com/hatchify/git-tagger
```
Comment (dhalman) [inline bin/init:6]: @itsmontoya fixed init script :)

**Review by itsmontoya — APPROVED** [review]

**Pull request merged**